pages/index.tsx

```diff
@@ -1,11 +1,16 @@
 import Image from "next/image";
 import HeroSection from "@/components/organisms/HeroSection";
+import TeamSection from "@/components/organisms/TeamSection"
 import Layout from "@/components/layouts";
+develop
```

Comment:
@Timimah this is what is causing this PR deployment to fail on netlify can you remove this?